test/test_model.py

```diff
@@ -47,50 +47,51 @@ def test_trivial_model():
 def test_average_model():
     """Check the implementation of the average DW model."""
 
+    data = np.ones((100, 100, 100, 6), dtype=float)
+
     gtab = np.array(
         [
+            [0, 0, 0, 0],
             [-0.31, 0.933, 0.785, 25],
             [0.25, 0.565, 0.21, 500],
             [-0.861, -0.464, 0.564, 1000],
             [0.307, -0.766, 0.677, 1000],
-            [0.736, 0.013, 0.774, 2000],
+            [0.736, 0.013, 0.774, 1300],
         ]
     )
 
-    gtab_w25 = gtab[1:, :]
-    gtab_1000 = gtab[2:3, :]
-    gtab_2000 = gtab[2:, :]
+    data *= gtab[:, -1]
 
     tmodel_mean = model.AverageDWModel(gtab=gtab, bias=False, stat="mean")
     tmodel_median = model.AverageDWModel(gtab=gtab, bias=False, stat="median")
     tmodel_1000 = model.AverageDWModel(
-        gtab=gtab, bias=False, th_high=1000, th_low=1000
+        gtab=gtab, bias=False, th_high=1000, th_low=900
     )
     tmodel_2000 = model.AverageDWModel(
-        gtab=gtab, bias=False, th_high=2000, th_low=1000
+        gtab=gtab, bias=False, th_high=2000, th_low=900, stat="mean",
```

Comment:
Ha! I just noticed that the default statistic for the "mean" model is the median. Isn't that a bit confusing?